cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -155,12 +156,13 @@ public void run() {
                         dismiss();
```

Comment:
Strangely yes. When I checked the flaky test failing, it always pointing the line 176 which is mPopup.dismiss().